scripts/doc-checker.py

```diff
@@ -11,6 +11,9 @@
 import os
 import ast
 from typing import List
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
```

Comment:
Hmm, it now came to my mind that this is also wrong... 😆 
As it would test Modin in which this script lives, but not what we asked it to test.

So the "real" fix should be assuring that numpydoc validator is importing the correct things, so we either need to modify `sys.path` according to files being checked or use an import hook.

Or at least validate that files we check live under this script's root repo (and transform paths to them accordingly when composing `module_name` variable down below).

Example that would fail in current version of the script: `python doc-checker.py ../../../modin-dev-2/modin/pandas/dataframe.py` (i.e. we can go outside of the repo that holds this script, and it would break).

Reply:
added check for failed case